### PR TITLE
Main table view basic styling

### PR DESCRIPTION
**Pull request by Flight**
- main table view basic styling
- main navigation
- main dropdowns

### CODE REVIEW

File: filer/private/sass/layout/_main.scss [hunk 1]
```diff
@@ -178,8 +188,155 @@
...
 @media screen and (max-width: 768px) {
     .image-info {
-        margin-top: 60px;
         float: none;
         width: 100%;
+        margin-top: 60px;
     }
 }
+
+body .filter-files {
```
Comment (FinalAngel) [inline filer/private/sass/layout/_main.scss]: do we need to use body here?

File: filer/private/sass/libs/_bootstrap.min.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/*!
```
Comment (FinalAngel) [inline filer/private/sass/libs/_bootstrap.min.scss]: why do we need to load the full bootstrap here?

File: filer/templates/admin/filer/base_site.html [hunk 3]
```diff
@@ -15,6 +15,7 @@
     <script src="{% static 'filer/js/libs/class.min.js' %}"></script>
     <script src="{% static 'filer/js/libs/jquery.min.js' %}"></script>
     <script src="{% static 'filer/js/libs/jquery-ui.min.js' %}"></script>
+    <script src="{% static 'filer/js/libs/bootstrap.min.js' %}"></script>
```
Comment (FinalAngel) [inline filer/templates/admin/filer/base_site.html]: I'm not a fan of adding bootstrap here

File: filer/private/sass/libs/_bootstrap.min.scss (hunk 2 above)
Comment (Flight) [inline filer/private/sass/libs/_bootstrap.min.scss]: It's not full, just grid system and dropdown menus.

File: filer/templates/admin/filer/base_site.html (hunk 3 above)
Comment (Flight) [inline filer/templates/admin/filer/base_site.html]: It's only for dropdown menu.

Comment (FinalAngel) [inline filer/templates/admin/filer/base_site.html]: than I suggest naming it bootstrap.custom.min

File: filer/private/sass/libs/_bootstrap.min.scss (hunk 2 above)
Comment (FinalAngel) [inline filer/private/sass/libs/_bootstrap.min.scss]: than I suggest naming it bootstrap.custom.min

File: filer/private/sass/layout/_main.scss [hunk 4]
```diff
@@ -1,21 +1,31 @@
...
+    min-width: 320px;
+}
 .text-left {
     text-align: left;
 }
 .text-right {
     text-align: right;
 }
+.clearfix:after {
+    content: '';
```
Comment (FinalAngel) [inline filer/private/sass/layout/_main.scss]: double quotes

File: filer/private/sass/layout/_main.scss [hunk 5]
```diff
@@ -1,21 +1,31 @@
...
     width: 28%;
     margin-top: 0;
-    border-radius: 3px;
     border: 0;
-    box-shadow: 0 0 5px 0 rgba(0,0,0,0.2);
-    background: #FFF;
-    box-sizing: border-box;
+    border-radius: 3px;
+    background: #fff;
+    box-shadow: 0 0 5px 0 rgba(0,0,0,.2);
```
Comment (FinalAngel) [inline filer/private/sass/layout/_main.scss]: ordering

File: filer/private/sass/layout/_main.scss [hunk 6]
```diff
@@ -56,9 +66,9 @@
                 margin-right: 15px;
             }
             li {
-                font-weight: normal !important;
                 padding: 0 !important;
                 border: 0 !important;
+                font-weight: normal !important;
```
Comment (FinalAngel) [inline filer/private/sass/layout/_main.scss]: ordering

File: filer/private/sass/layout/_main.scss [hunk 7]
```diff
@@ -178,8 +188,154 @@
...
+}
+
+.filter-files {
+    box-sizing: border-box;
+    width: 100%;
+    margin-bottom: 10px;
+    padding: 10px;
+    border: 1px solid #ddd;
+    outline: none;
+    font-size: 14px;
```
Comment (FinalAngel) [inline filer/private/sass/layout/_main.scss]: ordering, please check generally

File: filer/private/sass/layout/_main.scss [hunk 8]
```diff
@@ -178,8 +188,154 @@
...
+.navigator-top-nav {
+    clear: both;
+    padding: 10px 0;
+    .nav-button, .separator {
+        display: inline-block;
+        margin: 0 10px;
+        font-size: 26px;
+        vertical-align: top;
+        color: #555;
+        &.nav-button-filter {
```
Comment (FinalAngel) [inline filer/private/sass/layout/_main.scss]: do we need to nest this?

File: filer/private/sass/layout/_main.scss [hunk 9]
```diff
@@ -178,8 +188,154 @@
...
+        top: -1px;
+        width: 1px;
+        height: 26px;
+        margin: 0 10px;
+        background-color: #555;
+    }
+}
+.navigator-tools {
+    margin-top: 10px;
+    .navigator-button {
```
Comment (FinalAngel) [inline filer/private/sass/layout/_main.scss]: nesting required?

File: filer/private/sass/layout/_main.scss [hunk 10]
```diff
@@ -178,8 +188,154 @@
...
+    .navigator-button + .dropdown-menu {
+        margin-top: -2px;
+    }
+}
+
+.dropdown-container {
+    display: inline-block;
+    position: relative;
+    vertical-align: top;
+    &.dropdown-container-down {
```
Comment (FinalAngel) [inline filer/private/sass/layout/_main.scss]: please check nesting levels ;)

File: filer/templates/admin/filer/folder/directory_listing.html [hunk 11]
```diff
@@ -43,60 +42,122 @@
...
+                    {% else %}
+                        <a class="backtoroot" href="{% url 'admin:filer-directory_listing-root' %}{% if is_popup %}?_popup=1{% if select_folder %}&amp;select_folder=1{% endif %}{% endif %}" title="{% trans 'Go back to' %} {% trans 'root'|title %} {% trans 'folder' %}">&uarr;</a>
+                    {% endif %}
+                    <img src="{{ folder.icons.32 }}" alt="">
+                    {{ folder.name }}
+                    <span class="small quiet">
+                        ({% blocktrans count folder.children_count as counter %}{{ counter }} folder{% plural %}{{ counter }} folders{% endblocktrans %}, {% blocktrans count folder.file_count as counter %}{{ counter }} file{% plural %}{{ counter }} files{% endblocktrans %})
+                    </span>
+
+                    <div class="dropdown-container dropdown-container-down">
```
Comment (FinalAngel) [inline filer/templates/admin/filer/folder/directory_listing.html]: does the dropdown need to be in the h1?

File: filer/templates/admin/filer/folder/directory_listing.html [hunk 12]
```diff
@@ -43,60 +42,122 @@
...
+                <a href="#" class="nav-button nav-button-filter">
+                    <span class="fa fa-filter"></span>
+                </a>
+                <span class="separator"></span>
+                <div class="dropdown-container">
+                    <a href="#" data-toggle="dropdown" aria-expanded="false" class="nav-button nav-button-dots">
+                        <span class="fa fa-ellipsis-v"></span>
+                    </a>
+                    <ul class="create-menu-dropdown dropdown-menu">
+                        <li><a href="#">{% trans 'Delete selected files and/or folders' %}</a></li>
```
Comment (FinalAngel) [inline filer/templates/admin/filer/folder/directory_listing.html]: double quotes

File: filer/templates/admin/filer/folder/directory_listing.html [hunk 13]
```diff
@@ -43,60 +42,122 @@
...
     </div>
     <div id="content-main">
+        <input type="text" placeholder="Filter files" class="filter-files">
+
         <div class="js-navigator navigator{% if not actions_on_top and not actions_on_bottom %}navigator-no-actions{% endif %}">
-            {% include "admin/filer/tools/search_form.html" %}
-            <form class="js-navigator-form" action="" method="post">{% csrf_token %}
-                {% if is_popup %}<input type="hidden" name="_popup" value="1" />{% endif %}
-                {% if action_form and actions_on_top and paginator.count and not select_folder and not is_popup %}{% filer_actions %}{% endif %}
+            <form class="js-navigator-form" action="" method="post">
```
Comment (FinalAngel) [inline filer/templates/admin/filer/folder/directory_listing.html]: remove action if empty

**Comment by FinalAngel** [comment]
LGTM, would appreciate a second opinion though.

Also as a hint, when adding new commits, stop the old travis build to speed up the queue :)

**Comment by Brankub** [comment]
good work guys :+1:

**Pull request merged**